app.go

```diff
@@ -603,6 +608,23 @@ func New(config ...Config) *App {
 	// Override colors
 	app.config.ColorScheme = defaultColors(app.config.ColorScheme)
 
+	// If the app is configured to use services,
+	// register a post shutdown hook to shutdown them after the server is closed.
+	if app.hasServices() {
+		// TODO: support for custom context
+		ctx := context.Background()
+		if err := app.startServices(ctx); err != nil {
+			log.Warnf("failed to start services: %v", err)
```

Comment:
Should we panic() here?

Reply:
I did not consider panicking here because of the dev/test mode mindset, but it's probably fine: it the user defines services, they are expected to be healthy

Will change it asap

Reply:
Mmm, if we panic in the middle of the startup of the services, would the postShutdownHook be called? If not, then we can leak services in the running state